### ISSUE TITLE
How to run tests when forked repo?

### ISSUE THREAD

**Issue by edwvee** [hybridgroup/gocv#231]
I have forked this repo, run ```go get github.com/edwvee/opencv```, checked out dev branch, created my feature branch. Now I need to write some tests. But when I run ```go test github.com\edwvee\gocv``` I receive:
package github.com/edwvee/gocv: code in directory gopath\src\github.com\edwvee\gocv expects import "gocv.io/x/gocv"
As said on page https://github.com/hybridgroup/gocv/blob/master/CONTRIBUTING.md, I should run env.sh or env.cmd. These scripts are saying that I should run ```go run  -tags customenv ./cmd/version/main.go```. When I run this command I receive next output:
```
# gocv.io/x/gocv
In file included from core.cpp:1:0:
core.h:24:30: fatal error: opencv2/opencv.hpp: No such file or directory
compilation terminated.
```
I have installed opencv before, and without -tags customenv everything runs fine.
What should I do?

**Comment by deadprogram** [comment]
Please check out https://github.com/hybridgroup/gocv/blob/master/CONTRIBUTING.md#how-to-run-tests for this info.

**Comment by edwvee** [comment]
So, I did as said in this part of article:
```
ECHO See the Custom Environment section of the README if you need to customize your environment.
See the Custom Environment section of the README if you need to customize your environment.
```
Then  i ran ```go run -tags customenv ./cmd/version/main.go``` and received this:
```
# gocv.io/x/gocv
In file included from core.cpp:1:0:
core.h:24:30: fatal error: opencv2/opencv.hpp: No such file or directory
compilation terminated.
```

**Comment by deadprogram** [comment]
Normally you do not need to use `-tags customenv` unless you have installed OpenCV itself to a unique location.

In any case, you would normally just push your code to your own branch from `$GOPATH/src/gocv/x/gocv` no need to `go get` using your own separate repo.

**Comment by deadprogram** [comment]
I've removed the extra step from the contributing guidelines in the `dev` branch: https://github.com/hybridgroup/gocv/blob/dev/CONTRIBUTING.md#how-to-run-tests

I will close this issue after our next release.

**Comment by milosgajdos83** [comment]
What I do is I have multiple `git remote`s on the `github.com/hybridgroup/gocv` repo:

```shell
(dev)$ git remote -v
mine	https://github.com/milosgajdos83/gocv.git (fetch)
mine	https://github.com/milosgajdos83/gocv.git (push)
origin	https://github.com/hybridgroup/gocv (fetch)
origin	https://github.com/hybridgroup/gocv (push)
```

This is my workflow:

```shell
cd $GOPATH/gocv.io/x/gocv
git checkout dev && git checkout -b feature-branch
# do lots of crazy hacking; run tests the same way you'd do it if you had origin remote access
make test
# tests pass; push to my remote
git push mine feature-branch
# create PR
```

If I want my remote in sync with origin:

```shell
git checkout dev && git push mine -f
```

**Comment by deadprogram** [comment]
How about adding the git remote and checkout flow info to the contributing guide, @milosgajdos83 because that is useful info.

**Comment by milosgajdos83** [comment]
I would be happy to do that, I just never thought it would be useful to anyone besides me @deadprogram 

Will open a PR later on today.

**Comment by deadprogram** [comment]
These docs are now released as part of v0.14.0 so closing. Thank you!
